### PR TITLE
[Compute KeplerUpdate2][#38468 #39984] Update Defender injection

### PR DESCRIPTION
**Pull request by divyabhushan-okta**
## Description
Update Defender injection for java functions

* Maven and Gradle dependency updates.

### CODE REVIEW

**Comment by Pubs-MV** [comment]
Can you please review this @avish9

File: compute/admin_guide/install/install_defender/install_serverless_defender.adoc [hunk 1]
```diff
@@ -203,6 +203,38 @@ The steps for embedding Serverless Defender differ depending on the build tool.
...
+
+  mkdir -p com/twistlock/serverless/defender/22.06.286
+
+. Move the `twistlock*.jar` file into this new subdirectory.
+
+. Rename the *.jar file to the convention: defender-<version>.jar (e.g. defender-22.06.286.jar).
+
+. Create a file called defender-<version>-pom.xml in the same location of the jar (change the version tag based on your version):
+
+.. Define the internal (remote) repository in the `*pom.xml` file:
```
Comment (avish9) [inline compute/admin_guide/install/install_defender/install_serverless_defender.adoc]: Should be (local) as far as I understand

File: compute/admin_guide/install/install_defender/install_serverless_defender.adoc [hunk 2]
```diff
@@ -203,6 +203,38 @@ The steps for embedding Serverless Defender differ depending on the build tool.
...
+
+  <project>
```
Comment (avish9) [inline compute/admin_guide/install/install_defender/install_serverless_defender.adoc:218]: Don't see this mentioned in the GH issue

Comment (divyabhushan-okta) [reply compute/admin_guide/install/install_defender/install_serverless_defender.adoc:218]: This code comes from this [DR](https://spring.paloaltonetworks.com/twistlock/twistlock/wiki/%5BDR%5D-%5BServerless-Defender%5D-Defender-injection-for-java-functions#internal-repository). Please suggest! 
Thanks

Comment (chenBenjamin97) [reply compute/admin_guide/install/install_defender/install_serverless_defender.adoc:218]: The DR suggests a workaround for customers with older console versions (before Lagrange this feature was broken).
For customers using Lagrange - we fixed it in the BE, these manual steps are not required.

**Pull request merged**